contributors/devel/api-conventions.md

```diff
@@ -379,10 +381,15 @@ objects, with possibly a reference from the original object. This helps to
 ensure that GETs and watch remain reasonably efficient for the majority of
 clients, which may not need that data.
 
-Some resources report the `observedGeneration`, which is the `generation` most
-recently observed by the component responsible for acting upon changes to the
-desired state of the resource. This can be used, for instance, to ensure that
-the reported status reflects the most recent desired status.
+Sometimes not every aspect of an object's status can be modeled as a
```

Comment:
Generally speaking, I expect to see status API fields used when the information is structured and will be used for taking other programmatic action.  For example, the number of replicas in various states on the replicaset status or the acceptednames on the customresource status.